### PR TITLE
Adjust code style and lints to Raiden defaults

### PR DESCRIPTION
**Pull request by palango**

### CODE REVIEW

File: pathfinder/model/channel_view.py [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
 
 class ChannelView:
     """
     Unidirectional view of a bidirectional channel.
     """
     class State(Enum):
-        OPEN = 1,
-        SETTLING = 2,
+        OPEN = 1
+        SETTLING = 2
```
Comment (Dominik1999) [inline pathfinder/model/channel_view.py]: Why did you remove these commas here?

File: pathfinder/tests/fixtures/network_service.py [hunk 2]
```diff
@@ -1,17 +1,17 @@
 import random
-from typing import List, Callable, Generator
-
+from typing import Callable, Generator, List
 from unittest.mock import Mock, patch
+
```
Comment (Dominik1999) [inline pathfinder/tests/fixtures/network_service.py]: why do you need a new space here?

File: pathfinder/tests/test_blockchain_integration.py [hunk 3]
```diff
@@ -76,14 +75,14 @@ def test_pfs_with_mocked_client(
...
+        _p1_transferred_amount,
+        _p1_fee,
+        _p2_index,
         p2_deposit,
-        p2_transferred_amount,
-        p2_fee
+        _p2_transferred_amount,
+        _p2_fee,
     ) in enumerate(channel_descriptions_case_1):
         channel_identifier = channel_identifiers[index]
```
Comment (Dominik1999) [inline pathfinder/tests/test_blockchain_integration.py]: don't you also need a comma here then?

File: pathfinder/tests/test_graphs.py [hunk 4]
```diff
@@ -1,28 +1,28 @@
...
+from pathfinder.model import ChannelView, TokenNetwork
+from raiden_libs.types import Address
 
 
 def test_routing_benchmark(
     token_network_model: TokenNetwork,
-    populate_token_network_random: None
+    populate_token_network_random: None,
 ):
     value = 100
```
Comment (Dominik1999) [inline pathfinder/tests/test_graphs.py]: ... and also add a comma here? Or did the linter already tell you where to put them?

File: pathfinder/model/channel_view.py (hunk 1 above)
Comment (palango) [reply pathfinder/model/channel_view.py]: That was a mistake before, in enums no commas are needed.

File: pathfinder/tests/fixtures/network_service.py (hunk 2 above)
Comment (palango) [reply pathfinder/tests/fixtures/network_service.py]: `isort` adds a new line between python packages and third party packages.

File: pathfinder/tests/test_blockchain_integration.py (hunk 3 above)
Comment (palango) [reply pathfinder/tests/test_blockchain_integration.py]: Why? this is just a statement

File: pathfinder/tests/test_graphs.py (hunk 4 above)
Comment (palango) [reply pathfinder/tests/test_graphs.py]: Why? this is just a statement

Comment (LefterisJP) [reply pathfinder/tests/test_graphs.py]: (the linter tells you where to put the commas if you have the `flake8-commas` extension. And yes as paul said statements don't need commas. If you add a comma there you will get ... a tuple ;)

File: pathfinder/model/channel_view.py (hunk 1 above)
Comment (Dominik1999) [reply pathfinder/model/channel_view.py]: ah ok safe, thank you

File: pathfinder/tests/fixtures/network_service.py (hunk 2 above)
Comment (Dominik1999) [reply pathfinder/tests/fixtures/network_service.py]: ah ok thanks got it

File: pathfinder/tests/test_blockchain_integration.py (hunk 3 above)
Comment (Dominik1999) [reply pathfinder/tests/test_blockchain_integration.py]: ok so no commas behind statements but behind arguments, got it

File: pathfinder/tests/test_graphs.py (hunk 4 above)
Comment (Dominik1999) [reply pathfinder/tests/test_graphs.py]: good old linter, thanks!
